docs/readthedocs/tutorials/collisions.md

```diff
@@ -393,7 +392,7 @@ the creation function:
 
 ```cpp
 SoftBodyNode* bn = chain->createJointAndBodyNodePair<JointType, SoftBodyNode>(
-      parent, joint_properties, body_properties).second;
+      parent, properties, body_properties).second;
```

Comment:
```suggestion
      parent, joint_properties, body_properties).second;
```

Reply:
The variable name is reasonable. However, the write-up suggests people cut and paste the code from a previous function, which used `properties` instead of `joint_properties`. How about we use `joint_properties` throughout?